release/ray_release/test_automation/state_machine.py

```diff
@@ -249,3 +252,23 @@ def _keep_github_issue_open(self) -> None:
             "Re-opening issue as test is still failing. "
             f"Latest run: {self.test_results[0].url}"
         )
+
+    def comment_blamed_commit_on_github_issue(self) -> None:
+        """
+        Comment the blamed commit on the github issue.
+        """
+        blamed_commit = self.test.get(Test.KEY_BISECT_BLAMED_COMMIT)
+        issue_number = self.test.get(Test.KEY_GITHUB_ISSUE_NUMBER)
+        bisect_build_number = self.test.get(Test.KEY_BISECT_BUILD_NUMBER)
+        if not issue_number or not bisect_build_number or not blamed_commit:
+            logger.info(
+                "Skip commenting blamed commit on github issue "
+                f"for {self.test.get_name()}"
```

Comment:
maybe be more specific on the skipping reason?

in what case that the `blamed_commit`, `issue_number`, `bisect_build_number` can be missing or empty? is that an expected case or an unexpected error?

Reply:
this is a public function so it tries to be soft failed; it's an expectation from the caller that these three fields should exist

maybe i can return a boolean so the caller can handle the errors if needed